android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/home/views/HomeSurveysView.kt

```diff
@@ -0,0 +1,223 @@
+package co.nimblehq.ic.kmm.suv.android.ui.screens.home.views
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.gestures.Orientation
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.R
+import co.nimblehq.ic.kmm.suv.android.extension.placeholder
+import co.nimblehq.ic.kmm.suv.android.ui.components.DotsIndicator
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import coil.compose.AsyncImage
+import timber.log.Timber
+
+data class HomeSurveysUiModel(
+    val title: String,
+    val description: String,
+    val imageUrlString: String,
```

Comment:
Optional updates, I think we don't need the `String` in here, `imgeUrl` show its meaning imo 😄 Not a major one so you could leave it as it is.